_sass/charts.scss

```diff
@@ -3,11 +3,11 @@ rect.bar {
   mix-blend-mode: difference;
 }
 
-text {
-  fill: rgb(71, 71, 71);;
-  mix-blend-mode: luminosity;
-  font-family: 'Source Code Pro', monospace;
-}
+// text {
+//   fill: rgb(71, 71, 71);;
+//   mix-blend-mode: luminosity;
+//   font-family: 'Source Code Pro', monospace;
+// }
```

Comment:
```suggestion
text {
  fill: rgb(71, 71, 71);
  mix-blend-mode: luminosity;
  font-family: 'Source Code Pro', monospace;
  &.line-chart-label {
    mix-blend-mode: normal;
    font-size: 8px;
  }
}
```

Reply:
the above is to reduce the size of the line chart labels (which would be classed as such) and allowing the colours to show while retaining the source code pro font family. this only works if the line-chart-label class is given to the line chart label text (suggested below)